windows-agent/internal/proservices/wslinstance/wslinstance_test.go

```diff
@@ -95,7 +95,7 @@ func TestConnected(t *testing.T) {
 		ctx = wsl.WithMock(ctx, wslmock.New())
 	}
 
-	distroName, _ := wsltestutils.RegisterDistro(t, ctx, false)
+	defaultDistroName, _ := wsltestutils.RegisterDistro(t, ctx, false)
```

Comment:
This is a good change. Makes it clear that we are not copying the variable a few lines below just because of the old for-loop scope behavior.

Reply:
Yeah! I prefer we didn’t override it and tell at some point "oh, we forgot this" :p